test/rime_test_main.cc

```diff
@@ -1,10 +1,22 @@
 #include <gtest/gtest.h>
 #include <rime_api.h>
 #include <rime/service.h>
+#include <rime/module.h>
 #include <rime/setup.h>
 
+class GlobalEnvironment : public testing::Environment {
+ public:
+  void TearDown() override {
+    rime::Service::instance().deployer().JoinMaintenanceThread();
```

Comment:
These functions are not exported by rime.dll so the test program has link errors.

We need to add `RIME_DLL` declaration for them to be exported.

Alternatively, let's change the `SetUp` code to calling API functions `RimeSetup` and `RimeInitialize`, and `TearDown` code should call `RimeFinalize`.